pkg/controller.v1alpha3/suggestion/composer/composer.go

```diff
@@ -28,7 +28,7 @@ const (
 
 	cpuLimit   = "500m"
 	cpuRequest = "50m"
-	memLimit   = "100Mi"
+	memLimit   = "200Mi"
```

Comment:
I'd like that we can add resource field in `katib-config` suggestion data to make suggestion resource configurable for each one

Reply:
Agree.  If resource field is added in katib-config for any suggestion algoroithm, it will override the default settings.

Reply:
Ok, I will add these values to `katib-config` in this PR.